lib/monitoring/aws-lambda/LambdaFunctionMonitoring.ts

```diff
@@ -558,6 +572,24 @@ export class LambdaFunctionMonitoring extends Monitoring {
     props.useCreatedAlarms?.consume(this.createdAlarms());
   }
 
+  private convertToLatencyThreshold(
+    latencyProps: LatencyThreshold | LatencyTimeoutPercentageThreshold,
+    lambdaFunction: IFunction,
+  ): LatencyThreshold {
+    if ("maxLatency" in latencyProps) {
+      return latencyProps;
+    }
+    const { maxLatencyPercentage, ...rest } = latencyProps;
+    const timeout =
+      (lambdaFunction as Function).timeout ?? Duration.seconds(30);
```

Comment:
The default is 3 seconds.

https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_lambda.Function.html#timeout